overlay/tun_water_windows.go

```diff
@@ -52,11 +52,13 @@ func newWaterTun(c *config.C, l *logrus.Logger, cidr netip.Prefix, _ bool) (*wat
 
 func (t *waterTun) Activate() error {
 	var err error
+	//TODO multi-ip support
```

Comment:
this is a very ugly hack

Reply:
We can probably safely remove the fallback windows tap driver support now.